libs/designer/src/lib/ui/panel/panelroot.tsx

```diff
@@ -305,43 +318,52 @@ export const PanelRoot = (props: PanelRootProps): JSX.Element => {
-      panelScope={PanelScope.CardLevel}
-      panelHeaderControlType={getPanelHeaderControlType() ? PanelHeaderControlType.DISMISS_BUTTON : PanelHeaderControlType.MENU}
-      panelHeaderMenu={getPanelHeaderMenu()}
-      selectedTab={selectedPanelTab}
-      showCommentBox={showCommentBox}
-      tabs={registeredTabs}
-      nodeId={selectedNode}
-      onDismissButtonClicked={handleDelete}
-      readOnlyMode={readOnly}
-      setSelectedTab={setSelectedTab}
-      toggleCollapse={() => {
-        //only run validation when collapsing the panel
-        if (!collapsed) {
-          Object.keys(inputs?.parameterGroups ?? {}).forEach((parameterGroup) => {
-            inputs.parameterGroups[parameterGroup].parameters.forEach((parameter) => {
-              const validationErrors = validateParameter(parameter, parameter.value);
-              dispatch(
-                updateParameterValidation({ nodeId: selectedNode, groupId: parameterGroup, parameterId: parameter.id, validationErrors })
-              );
+    <>
+      <PanelContainer
+        {...commonPanelProps}
+        cardIcon={iconUri}
+        comment={comment}
+        noNodeSelected={!selectedNode}
+        isError={errorInfo?.level === ErrorLevel.Critical || opQuery?.isError}
+        errorMessage={errorInfo?.message}
+        isLoading={isLoading}
+        panelScope={PanelScope.CardLevel}
+        panelHeaderControlType={getPanelHeaderControlType() ? PanelHeaderControlType.DISMISS_BUTTON : PanelHeaderControlType.MENU}
+        panelHeaderMenu={getPanelHeaderMenu()}
+        selectedTab={selectedPanelTab}
+        showCommentBox={showCommentBox}
+        tabs={registeredTabs}
+        nodeId={selectedNode}
+        onDismissButtonClicked={handleDelete}
+        readOnlyMode={readOnly}
+        setSelectedTab={setSelectedTab}
+        toggleCollapse={() => {
+          //only run validation when collapsing the panel
+          if (!collapsed) {
+            Object.keys(inputs?.parameterGroups ?? {}).forEach((parameterGroup) => {
+              inputs.parameterGroups[parameterGroup].parameters.forEach((parameter) => {
+                const validationErrors = validateParameter(parameter, parameter.value);
+                dispatch(
+                  updateParameterValidation({ nodeId: selectedNode, groupId: parameterGroup, parameterId: parameter.id, validationErrors })
+                );
+              });
             });
-          });
-        }
-        togglePanel();
-      }}
-      trackEvent={handleTrackEvent}
-      onCommentChange={onCommentChange}
-      title={selectedNodeDisplayName}
-      onTitleChange={onTitleChange}
-    />
+          }
+          togglePanel();
+        }}
+        trackEvent={handleTrackEvent}
+        onCommentChange={onCommentChange}
+        title={selectedNodeDisplayName}
+        onTitleChange={onTitleChange}
+      />
+      <DeleteNodeModal
+        nodeId={selectedNode}
+        nodeType={WORKFLOW_NODE_TYPES.GRAPH_NODE}
```

Comment:
Make sure this type value is updated properly so the delete modal renders content with respect to the node